docs/best_practices/python.md

```diff
@@ -0,0 +1,167 @@
+# Python best practices
+
+## Import
+
+### Avoid `from module_foo import symbol_bar`
```

Comment:
Just realized, this one is not applied in the current `buildchain` (I see a lot of `from buildchain import foo` statements).

Reply:
Indeed ^^"

That's a proof that I didn't only write stuff I already apply myself :p

We can fix it when we add unit test for the buildchain, as it will interfer with the mocking (or we can make a followup PR just for that).

Reply:
Let's wait until we add tests/mocking